docs/user/dql/functions.rst

```diff
@@ -234,9 +234,63 @@ CEIL
 Description
 >>>>>>>>>>>
 
-Specifications:
+Usage: CEIL(a) takes the ceiling of value a.
+
+Limitation: CEIL only works as expected when IEEE 754 double type displays decimal when stored.
+
+Argument type: a: double
+
+Return type: LONG
+
+Example::
+
+   os> SELECT CEIL(0), CEIL(50.00005), CEIL(-50.00005);
+    fetched rows / total rows = 1/1
+    +-----------+------------------+-------------------+
+    | CEIL(0)   | CEIL(50.00005)   | CEIL(-50.00005)   |
+    |-----------+------------------+-------------------|
+    | 0         | 51               | -50               |
+    +-----------+------------------+-------------------+
+
+   os> SELECT CEIL(3147483647.12345), CEIL(113147483647.12345), CEIL(3147483647.00001);
+    fetched rows / total rows = 1/1
+    +--------------------------+----------------------------+--------------------------+
+    | CEIL(3147483647.12345)   | CEIL(113147483647.12345)   | CEIL(3147483647.00001)   |
+    |--------------------------+----------------------------+--------------------------|
+    | 3147483648               | 113147483648               | 3147483648               |
+    +--------------------------+----------------------------+--------------------------+
+
+CEILING
+----
+
+Description
+>>>>>>>>>>>
+
+Usage: CEILING(a) takes the ceiling of value a.
```

Comment:
Can you add cross links between `CEIL` and `CEILING`?

Reply:
Addressed (I think) in b5fd1208eb4d4012ce8a7fa16b93a2b50ac4eaf1